FestivalPlanner/templates/screenings/details.html

```diff
@@ -24,11 +26,11 @@ <h3>Screening details</h3>
     <li>Time: {{ screening.start_dt|date:"D j M H:i" }} - {{ screening.end_dt|date:"H:i" }}</li>
     <li>Film details: <a href="{% url 'films:details' screening.film.id %}">{{ screening.film.title }} details</a></li>
     <li>Film website: <a href="{{ screening.film.url }}">{{ screening.film.title }}</a></li>
-    <li>Film description: {{ film_description }}</li>
+    <li>Film description: {{ film_description|default_if_none:" - " }}</li>
     <li>Screening duration: {{ duration }} ({{ minutes }})</li>
-    <li>Subtitles: {% if screening.subtitles %} {{ screening.subtitles }} {% else %} - {% endif %}</li>
-    <li>Q&A: {% if screening.q_and_a %} {{ screening.q_and_a }} {% else %} - {% endif %}</li>
-    <li>Combination program: {% if screening.combination_program %} {{ screening.combination_program }} {% else %} - {% endif %}</li>
+    <li>Subtitles: {{ screening.subtitles|default_if_none:" - " }}</li>
```

Comment:
`screening.subtitles` doesn't seem to be `None`.
Please add `null=True` to the field in the model.

Reply:
Done.